data/LARA/projects.yaml

```diff
@@ -123,10 +123,11 @@ projects:
       type: Lab GitHub
       url: https://github.com/epfl-lara/rust-stainless
       date_last_commit: 2021-07-22
+    maturity: 2
```

Comment:
I thought we convinced ourselves it's 1, no?

Reply:
After rerereading the document, yep, you're right